lang/zh.json

```diff
@@ -153,7 +153,7 @@
             "label": "长征探索任务",
             "fields": {
               "complete": "已完成探索",
-              "freeComplete": "Free Expedition Complete",
+              "freeComplete": "自由探索已完成",
```

Comment:
These aren't consistent with eachother, but they're basically the same thing, but one is for free expeditions

Reply:
What do you mean? Are there still paid tasks?